.eslintrc

```diff
@@ -7,7 +7,7 @@
     "jest": true
   },
   "rules": {
-    "@typescript-eslint/ban-ts-ignore": "warn",
+    "@typescript-eslint/ban-ts-comment": "warn",
```

Comment:
Why not add this new rule?

Reply:
ban-ts-ignore was replaced by ban-ts-comment. 
https://github.com/typescript-eslint/typescript-eslint/releases/tag/v3.0.0